hbt/selection/jet.py

```diff
@@ -32,7 +32,7 @@
     },
     produces={
         # new columns
-        "Jet.hhbtag",
+        "Jet.hhbtag", hhbtag,
```

Comment:
It is already saved as Jet quantity, why do you want to save it a second time?
```suggestion
        "Jet.hhbtag",
```